### ISSUE TITLE
SC 412 -- Text and email buttons have unnecessary aria-describedby

### ISSUE THREAD

**Issue by damiansian** [newjersey/accessibility-pm#17]
## URL
https://demo.fileyourstatetaxes.org/en/questions/contact-preference

## Issue
The buttons for "Text me a code" and "Email me a code" have `aria-describedby` to associate the adjacent heading. The heading is getting read after each button which is confusing. 

> [!Note]
> If this component had two buttons of "Yes" and "No", having the `aria-describedby` associate the adjacent heading would add value. In this case, since the accessible name of each control is explicit, this step just adds noise to the system.

![Image](https://github.com/user-attachments/assets/52175f57-91df-42cf-95cd-e9d2e0a78cd9)

## WCAG Success Criterion
[SC 4.1.2: Name, Role, Value (Level A)](https://w3.org/TR/WCAG21#name-role-value)

## User Impact
A screen reader using these controls is getting additional semantics that offer no real value. 

### Current computed accessible name is: 
> Text me a code, Next, create your account with a quick code

```html
<button 
    name="state_file_contact_preference_form[contact_preference]" 
    type="submit" id="state_file_contact_preference_form_submit" 
    value="text" class="button" 
    aria-describedby="main-question">
Text me a code
</button>
``` 

### Should be:
> Text me a code

```html
<button 
    name="state_file_contact_preference_form[contact_preference]" 
    type="submit" id="state_file_contact_preference_form_submit" 
    value="text" class="button" 
Text me a code
</button>
```

**Comment by damiansian** [comment]
Looks good. 

<img width="1364" alt="Described by is removed. " src="https://github.com/user-attachments/assets/583ea497-bae7-49b1-a027-7750aa8de1e8" />
